.ci-builds/.azure-pipelines.yml

```diff
@@ -134,7 +134,29 @@ jobs:
         CC: clang
         CXX: clang++
         LANG: "en_US@calendar=gregorian;hours=h12"
-
+#-------------------------------------------------------------------------
+- job: ICU4C_Clang13_Cpp20_Ubuntu_2004
+  displayName: 'C: Linux Clang-13 C++20 (Ubuntu 20.04)'
+  timeoutInMinutes: 30
+  pool:
+    vmImage: 'ubuntu-20.04'
+  steps:
+    - checkout: self
+      lfs: true
+      fetchDepth: 10
+    # Install Clang-13 from https://apt.llvm.org/
+    - script: |
+        curl -Ls https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+        sudo apt-add-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-13 main'
+        sudo apt update
+        sudo apt install -qy clang-13 lld-13 libc++-13-dev libc++abi-13-dev
+      displayName: 'Install Clang-13'
+    - script: |
+        export CXXFLAGS="-std=c++20" && cd icu4c/source && ./runConfigureICU Linux && make -j2 check
```

Comment:
Why is `CXXFLAGS` not in the `env` section together with `CC` and `CXX`?

Reply:
Ah, the other build bots in this config file used `export` for the `CXXFLAGS`, so I did it here to be consistent.
However, I think you're right that it might be better to do it the same as `CC` and `CX`. 
I'll push an update in a moment.